bindings/aws/sqs/sqs.go

```diff
@@ -60,15 +61,19 @@ func (a *AWSSQS) Init(metadata bindings.Metadata) error {
 		return err
 	}
 
-	queueName := m.QueueName
-	resultURL, err := client.GetQueueUrl(&sqs.GetQueueUrlInput{
-		QueueName: aws.String(queueName),
-	})
-	if err != nil {
-		return err
-	}
+	if m.QueueURL != "" {
```

Comment:
@amimimor can you please review the SQS portion regarding the addition of `queueURL`? Thanks!

Reply:
Sorry to spoil a little here:
I don't see a lot of value passing in the full queue URL as it can be built based on the region+accountID+queueName so the call to AWS SQS might be expensive here and could be replaced by some sorts of string builder (see: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-queue-message-identifiers.html)
(as with pubsub sns/sqs) it needs to be documented that using accesskey and secretkey need not to be used if using AWS node/pod roles
regarding the above @yaron2 waiting for your input

@yaron2 I think there is a lot to be done here in terms of using contexts, configured retries and durations and in general I would have felt more comfortable seeing that this component somehow aligns (I know it depends on community contribution...) with pubsub sns/sqs